docs/node/run-your-node/advanced/signing-key-manager-policy.md

```diff
@@ -0,0 +1,197 @@
+# Signing Key Manager Policy
+
+This guide will describe how to print and sign an Oasis key manager policy.
+
+:::info
+
+These instructions are only applicable if you are part of a key manager policy
+signer set.
+
+:::
+
+## Prerequisites
+
+### Oasis Node Binary
+
+Make sure you have followed the [Oasis Node binary installation guide] and have
+the Oasis Node binary installed on your system.
+
+### Entity
+
+Similarly to other things, an entity's private key is used to sign a key manager
+policy.
+
+The trusted key manager policy signer set is hard-coded in the key manager's
+source code.
+
+:::tip EXAMPLE
+
+The Oasis Key Manager's trusted policy signer set is defined [here
+][oasis-km-signer-set].
+
+:::
+
+:::danger
+
+We strongly recommend using a dedicated (single-purpose) entity for signing key
+manager policies for production key managers, i.e. the ones deployed on
+Mainnet and connected to a production ParaTime.
+
+:::
+
+
+To provision a new entity, follow the [instructions in our Validator Node
+guide].
+
+:::danger
+
+We strongly advise AGAINST using an entity that is generated with a file-based
```

Comment:
We're not using capitalization for emphasizing stuff, but rather bolding the text:

```suggestion
We strongly advise **against using an entity that is generated with a file-based
```

Reply:
> We're not using capitalization for emphasizing stuff, but rather bolding the text:

Ok, I've changed to bold.